packages/remark-mdx/block.js

```diff
@@ -26,7 +26,7 @@ const elementCloseExpression = /^$/
 const otherElementOpenExpression = new RegExp(openCloseTag.source + '\\s*$')
 
 function blockHtml(eat, value, silent) {
-  const blocks = '[a-z\\.]+(\\.){0,1}[a-z\\.]'
+  const blocks = '[a-z0-9\\.]+(\\.){0,1}[a-z0-9\\.]'
```

Comment:
Would this regex allow for pure numeric blocks like:
```html
<00/>
<0.0/>
```

?

Is that intended?

Reply:
Good catch, I've gone ahead and made the regex more thorough and added more test cases.